packages/platform/platform-fastify/test/test-session.integration.spec.ts

```
@@ -0,0 +1,117 @@
+      return Promise.reject(new NotFound("User not found"));
+    }
+
+    return Promise.resolve(user);
+  }
+
+  @Get("/logout")
+  @Returns(204)
+  logout(@Req() req: any) {
+    switch (this.platformName) {
+      case "koa":
+        return Promise.resolve((req.ctx.session = null));
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)